k-means-minus-two/script.whizzml

```diff
@@ -8,51 +8,69 @@
 ;; centroid distance, the list of l anomalous points, and a list of the
 ;; similarity between sets of anomalies at each step of the algorithm.
 
+(define (headers fields)
+  (let (sorted-fields (sort-by-key "column_number" (values fields)))
+    (map (lambda (f) (f "name")) sorted-fields)))
+
+(define (make-anomalies-ds clustering)
+  (let (a-sample-id (clustering 3)
```

Comment:
I would use a map instead of a list to avoid this magic numbers

Reply:
We can transform it into a list at the end, to generate the final output. I think this way is easier to extend and understand